### PR TITLE
Archive feature Part 2 and 3: Update List and Add command

### LINKED ISSUE #90: Archive Feature: Update List Command
list and list /r

### LINKED ISSUE #91: Archive Feature: Update add command

### PR DESCRIPTION
**Pull request by WeiJie96**
Close #90 
Close #91

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/exceptions/CommandException.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package seedu.address.logic.commands.exceptions;
 
+import seedu.address.logic.commands.Command;
```
Comment (ijavierja) [inline src/main/java/seedu/address/logic/commands/exceptions/CommandException.java:3]: Doesnt look like Command is used here. Why import it?

File: src/main/java/seedu/address/model/person/IsArchivePredicate.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.person;
```
Comment (ijavierja) [inline src/main/java/seedu/address/model/person/IsArchivePredicate.java:1]: I think its a little strange to put a predicate in the person folder. Perhaps we can package it in model with where it is used at or put the whole class into the Model class.

**Review by ijavierja — COMMENTED** [review]
Some minor stuff. otherwise LGTM

File: src/main/java/seedu/address/model/person/IsArchivePredicate.java (hunk 2 above)
Comment (WeiJie96) [reply src/main/java/seedu/address/model/person/IsArchivePredicate.java:1]: The predicate is meant for filtering the field in the Person, just like how we placed the Predicate for filtering Name in the same folder

File: src/main/java/seedu/address/logic/commands/exceptions/CommandException.java (hunk 1 above)
Comment (WeiJie96) [reply src/main/java/seedu/address/logic/commands/exceptions/CommandException.java:3]: It's a really minor thing, on line 6 it {@link Command} in the Javadoc, so if we dont import it, it wont display properly

**Review by ijavierja — APPROVED** [review]
LGTM

**Pull request merged**